packages/contracts/contracts/L2/L1Block.sol

```diff
@@ -60,9 +60,9 @@ contract L1Block is Semver {
     uint256 public l1FeeScalar;
 
     /**
-     * @notice The ratio to distribute transaction fees as validator reward. 4 decimal.
+     * @notice The scalar to distribute transaction fees as validator reward. The denominator is 10000, so the ratio is expressed in 4 decimal places.
```

Comment:
Max line length of a solidity file is forced to 100. See the [solhint setting](https://github.com/kroma-network/kroma/blob/dev/packages/contracts/.solhint.json#L8).

Reply:
Applied.